### PR TITLE
Don't panic when the error length is slightly over 100

### PR DESCRIPTION
**Pull request by onyxcherry**
# Pull Request

## What does this PR do?
Fixes PR #2207 as [the last commit](https://github.com/meilisearch/meilisearch/pull/2207/commits/7ece7a9d9ee3015a84f3b19f6b7360b7099b5388) has changed number of the characters at the end to leave in place from `50` to `85` **but the lower limit of a string length wasn't changed**.
Therefore, any data (e.g. example string from issue #2680) was causing `meilisearch` to **panic**.

So I simply raised the minimum value from `100` to `135` (`50 + 85`) to ensure that `replace_range()` won't panic due to an inverted range.
At the same time I am in favor of the `85` value which was changed in the @CNLHC's last commit.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing ~issue~ pull request?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Review by Kerollmops — CHANGES_REQUESTED** [review]
Wow, thank you very much for your work! Fixing a bug we weren't even aware of Ahah!
Would it be possible to declare both of the numbers in variables just above, please?

File: meilisearch-lib/src/document_formats.rs [hunk 1]
```diff
@@ -136,16 +143,23 @@ pub fn read_json(input: impl Read, writer: impl Write + Seek) -> Result<usize> {
...
     for object in content.inner.map_right(|o| vec![o]).into_inner() {
+        println!("{:#?}", object);
         builder
             .append_json_object(&object)
             .map_err(Into::into)
             .map_err(DocumentFormatError::Internal)?;
     }
 
     let count = builder.documents_count();
+    println!("{count}");
```
Comment (Kerollmops) [inline meilisearch-lib/src/document_formats.rs:161]: Would you please remove the `println`s?

**Review by Kerollmops — CHANGES_REQUESTED** [review]

File: meilisearch-lib/src/document_formats.rs (hunk 1 above)
Comment (onyxcherry) [reply meilisearch-lib/src/document_formats.rs]: Yes, obviously. 
Very sorry about that 😅

**Review by Kerollmops — APPROVED** [review]
Thank you very much, LGTM!
bors merge

**Pull request merged**